### PR TITLE
improve grammar of README

### PR DESCRIPTION
**Pull request by jonlaokan**
Pull request title

- Clearly and concisely describes what it does
- Refer to the issue that it solves, if available

Pull request description

- Describes the main changes that come with the pull request
- Any relevant additional information is provided

> For detailed contributing instructions see https://github.com/iluwatar/java-design-patterns/wiki/01.-How-to-contribute

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -35,7 +35,7 @@ Once you are familiar with these concepts you can start drilling down into
 patterns by any of the following approaches
 
  - Using difficulty tags, `Difficulty-Beginner`, `Difficulty-Intermediate` & `Difficulty-Expert`.
- - Using pattern categories, `Creational`, `Behavioral` and others.
+ - Using pattern categories, `Creational`, `Behavioral`,and others.
```
Comment (iluwatar) [inline README.md]: Shouldn't there be a space after the comma?

Comment (jonlaokan) [reply README.md]: Indeed, a space is needed.

**Comment by iluwatar** [comment]
Ping @jonlaokan

**Comment by iluwatar** [comment]
Thanks @jonlaokan, even small things are helpful :)

**Pull request merged**